src/test/regress/expected/multi_alter_table_add_constraints.out

```diff
@@ -488,3 +488,162 @@ SELECT "Constraint", "Definition" FROM table_checks WHERE relid='public.products
 
 \c - - - :master_port
 DROP TABLE products;
+SET citus.shard_count to 2;
+-- Test if the ALTER TABLE %s ADD %s PRIMARY KEY %s works
+CREATE SCHEMA sc1;
+CREATE TABLE sc1.alter_add_prim_key(x int, y int);
+CREATE UNIQUE INDEX CONCURRENTLY alter_pk_idx ON sc1.alter_add_prim_key(x);
+ALTER TABLE sc1.alter_add_prim_key ADD CONSTRAINT alter_pk_idx PRIMARY KEY USING INDEX alter_pk_idx;
+SELECT create_distributed_table('sc1.alter_add_prim_key', 'x');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+SELECT run_command_on_workers($$
+    SELECT  
+        kc.constraint_name 
+    FROM 
+        information_schema.table_constraints tc join information_schema.key_column_usage kc on (kc.table_name = tc.table_name and kc.table_schema = tc.table_schema and kc.constraint_name = tc.constraint_name)
+    WHERE
+        kc.table_schema = 'sc1' and tc.constraint_type = 'PRIMARY KEY' and kc.table_name LIKE 'alter_add_prim_key_%' 
+    ORDER BY
+        1
+    LIMIT 
+        1;
+    $$)
+ORDER BY 
+    1;
+          run_command_on_workers          
+------------------------------------------
+ (localhost,57637,t,alter_pk_idx_1450234)
+ (localhost,57638,t,alter_pk_idx_1450234)
+(2 rows)
+
+CREATE SCHEMA sc2;
```

Comment:
I think tests for `sc2` and `sc1` has no difference from Citus' perspective. Feel free to remove  `sc2` tests.